package.json

```diff
@@ -93,7 +93,7 @@
     "clean-e2e": "node ./scripts/cleanE2e.mjs",
     "crowdin:upload": "echo 'Uploading sources to Crowdin' && crowdin upload sources --config ./crowdin.yaml",
     "crowdin:download": "echo 'Downloading translations from Crowdin' && crowdin download --config ./crowdin.yaml",
-    "jest": "node ./packages/jest-cli/bin/jest.js",
+    "jest": "FORCE_COLOR=1 node ./packages/jest-cli/bin/jest.js",
```

Comment:
On CI the `test-ci-partial:parallel` script is used:

https://github.com/jestjs/jest/blob/0fd5b1c37555f485c56a6ad2d6b010a72204f9f6/.github/workflows/test.yml#L37

It includes the `--color` flag, so all should work without `FORCE_COLOR=1` adding:

https://github.com/jestjs/jest/blob/0fd5b1c37555f485c56a6ad2d6b010a72204f9f6/package.json#L107

Reply:
@mrazauskas This is what you said in [#14391 (comment)](https://github.com/jestjs/jest/issues/14391#issuecomment-1669339073):

> The trick is that `--colors` is not passed anywhere and does not exist in Jest’s logic at all. `chalk` simply checks if `--colors` (or any numerous variation of it) is present in `process.argv`.

I actually think it should stay this way. In my opinion, having `--color` and the like have the same effect as setting `FORCE_COLOR`, or introducing any other means to set the color level for both the framework and the tests at the same time would be a bad decision because the uses of colors in the two have completely different meanings.

One might want to test colored output of a program that can produce such output in an environment that does not support colored output at all, and that would be perfectly valid. One might also want to ignore output coloring in tests and only focus on the contents of the output in an environment that supports colored output, but then using `--no-color` for this would be strange because that would also disable colors in the framework output, which we don't want.

Actually, my original solution with the `FORCE_COLOR=1` prefix is also bad with respect to what I have just explained since it leaves no option to disable colors in the framework output.

Another problem I noticed is that if tests were to be run in an environment with no color support, then (almost?) all snapshot tests would fail because snapshots for the repo are apparently produced in an environment where there is such support, and the current behavior is essentially to simply let tests deduce the color level from the environment.

I solved both problems by adding a setup file to `jest.config.mjs` in which `FORCE_COLOR` is set to `'1'` for all tests. (The configuration is only for internal Jest testing, so #14391 would still be fixed by merging this PR.)